web-client/src/app/state/session-xrpl.service.ts

```diff
@@ -423,51 +462,70 @@ export class SessionXrplService {
 
   protected async prepareUnsignedTransaction(
     receiverId: string,
-    amount: xrpl.Payment['Amount']
+    amount: xrpl.Payment['Amount'],
+    sender?: string
   ): Promise<xrpl.Payment> {
     const { wallet } = this.sessionQuery.assumeActiveSession();
+    const senderId = sender ? sender : wallet.xrpl_account.address_base58;
```

Comment:
Should we move line 468 into 469?
`const senderId = sender ? sender : this.sessionQuery.assumeActiveSession().wallet.xrpl_account.address_base58;`

Reply:
I see you reverted the change, it didn't work the first time because you had to specify "wallet" when writing `assumeActiveSession().wallet.xrpl...`